### PR TITLE
Django static file loading & readme update

### LINKED ISSUE #95: Obsługa staticów przez django
## Jaki jest cel tego zadania
Obecnie po uruchomieniu `docker compose` pliki static nie ładują się. Dopiero po ponownym uruchomieniu `docker compose exec api python manage.py collectstatic` zaczynają działać. Celem tego taska jest skonfigurowanie dockera tak aby nie trzeba było za każdym uruchomieniem włączać collectstatic.

[Docsy django](https://docs.djangoproject.com/en/5.0/howto/static-files/)

### PR DESCRIPTION
**Pull request by Sygnator**
Setup django static file loading automatically with dockerfile.
Update project README.md, new good-looking version with beginner's guide.

closes #95

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,40 +1,86 @@
...
+
+    ```sh
+    git checkout -b {your_branch_name}
+    ```
+
+4. **Make sure you are working on the correct branch**:
+
+   ```sh
+    git status
+    ```
```
Comment (pgronkievitz) [inline README.md:40]: **nitpick:** I'd drop this bold, if everything's bolded - nothing is

File: README.md [hunk 2]
```diff
@@ -1,40 +1,86 @@
...
 
-#### Starting app
+3. **Starting app**:
 
-```sh
-docker compose --profile prod up
-```
+    ```sh
+    docker compose --profile prod up
+    ```
```
Comment (pgronkievitz) [inline README.md:86]: **suggestion:** I find headers perfectly fine. we may want to use both h4 and numbers to make it extra-explicit

File: README.md (hunk 1 above)
Comment (TheJimmyNowak) [reply README.md:40]: +1\

File: README.md [hunk 3]
```diff
@@ -1,40 +1,86 @@
...
+
+[![Python](https://img.shields.io/badge/Python-3.11%2B-blue)](https://www.python.org/)
+[![Docker](https://img.shields.io/badge/Docker-24.0%2B-blue)](https://www.docker.com/)
+[![Poetry](https://img.shields.io/badge/Poetry-1.6%2B-blue)](https://python-poetry.org/)
+[![Django](https://img.shields.io/badge/Django-3.2%2B-green)](https://www.djangoproject.com/)
+[![Discord.py](https://img.shields.io/badge/Discord.py-2.3.2%2B-blue)](https://discordpy.readthedocs.io/en/stable/)
+[![PostgreSQL](https://img.shields.io/badge/PostgreSQL-16%2B-blue)](https://www.postgresql.org/)
+[![Git](https://img.shields.io/badge/Git-2.40%2B-red)](https://git-scm.com/)
+[![MIT License](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+
```
Comment (TheJimmyNowak) [inline README.md:13]: **praise:** Badgeeeeeee. Dobry pomysły

**Review by TheJimmyNowak — APPROVED** [review]

**Review by pgronkievitz — APPROVED** [review]

**Pull request merged**